test/data_center/combiner/init_model.py

```diff
@@ -0,0 +1,59 @@
+import keras
```

Comment:
In the last version in master, I changed the way we do the seeding for the mnist example. Instead of trying to write directly to Minio, we now produce the model file  and bundle it in the repo for convenience. Can you do the same for this example?  (see test/mnist-multi-combiner/seed/ ) and the updated README.md.